packages/browser-repl/src/components/shell-output-line.tsx

```diff
@@ -48,6 +49,10 @@ export class ShellOutputLine extends Component<ShellOutputLineProps> {
       return <HelpOutput value={value} />;
     }
 
+    if (shellApiType === 'CommandResult') {
```

Comment:
I think the idea of `CommandResult` was a bit different. @aherlihy suggested to use it for all the commands that has pre-formatted output. So that would not work really well, if we want to do what you suggested we need a specific type that identifies the `ShowDbsOutput` something like `ShowDbsResult`.

Reply:
ah! i was not aware that this type of result was reserved for something specific. I'll change this particular one to `ShowDbsResult` then.